apps/search/js/providers/suggestions.js

```diff
@@ -59,6 +69,8 @@
     },
 
     search: function(input, preventRemote) {
+
+      this.currentSearch = input;
```

Comment:
I don't really like the idea of keeping a current search here, so let's remove it if possible. The problem is that we interact with the search input state outside of the provier, so it's easy to get out of sync. E.g., search for something, clear rocketbar, change provider. With this patch we would then show suggestions with an empty search term.

If keeping this is the best option, perhaps a quick fix would be to check to see if there is content that exists within the suggestions already? Skipping the search if there is no content?